### ISSUE TITLE
Test Stability review: Include Cypress Test failures as a blocking check

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#68376]
## Issue Description
When we run CI, the number of jobs that run in parallel for the standard Cypress Test run varies between 8 and 12 jobs depending on how many tests are being run. Due to this fluctuation, the last 4 jobs cannot be made required in GHA due to that if they don't run, and they're required, it will automatically block the branch. The result of it is the last 4 jobs will allow failing or flaky tests to be merged into main.

We can combat this by making the status of the overall cypress test job (gets a pass or fail based on all 12 parts) a part of our Test Stability Review check. This would prevent merges that have failure in those 4 parallel jobs.

---
## Tasks
- [ ] Add a step to the Test Stability Review Merge Status check in GHA to verify that regular cypress tests passed.

## Acceptance Criteria
- [ ] Test failures in processes 8-11 of the Cypress Tests job can no longer be merged.
---

**Comment by JoeTice** [comment]
Descriptions will be added shortly based on our discussions

**Comment by CBonade** [comment]
Primary work for this is done and just awaiting some testing to merge. The file that checks for merge eligibility has been reworked to allow all applicable error messages to be displayed.

**Comment by CBonade** [comment]
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeDErQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--17648972a18d0a6702dafab0c78489520b83f487/image.png)
